packages/react-icons/src/components/YammerIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import createSvgIcon from '../utils/createSvgIcon';
+
+const YammerIcon = createSvgIcon({
+  svg: ({ classNames }) => (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0,0,2048,2048">
+      <g className="thing">
```

Comment:
className="thing", also classNames doesn't seem to be used.

Reply:
I know this className is being used in an example, but this doesn't seem like a good pattern to demonstrate in general. Maybe find another way to re-style the element for the example?

Reply:
Ah, I just copied this examples from the `Icon.Svg.Example.tsx`. Will create new example with showcase of how the things inside the svgs can be styled

Reply:
Refactored and improved examples. Please take a look.